tests/python/unittest/test_gluon_data.py

```diff
@@ -269,6 +269,12 @@ def test_multi_worker_dataloader_release_pool():
     if os.name == 'nt':
         print('Skip for windows since spawn on windows is too expensive.')
         return
+
+    from sys import platform
+    if platform == 'darwin':
+        print('Skip for MacOS due to flaky failures')
```

Comment:
```suggestion
        print('Skip for MacOS due to https://github.com/apache/incubator-mxnet/issues/17782')
```

Reply:
@haojin2 can you merge the suggested change or something similar to address Marco's concern?